app/src/main/java/com/mapbox/maps/testapp/examples/NinePatchImageActivity.kt

```diff
@@ -0,0 +1,86 @@
+package com.mapbox.maps.testapp.examples
+
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Feature
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.image.image
+import com.mapbox.maps.extension.style.layers.generated.SymbolLayer
+import com.mapbox.maps.extension.style.layers.generated.symbolLayer
+import com.mapbox.maps.extension.style.layers.getLayer
+import com.mapbox.maps.extension.style.layers.properties.generated.IconTextFit
+import com.mapbox.maps.extension.style.sources.generated.geoJsonSource
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_add_marker_symbol.*
+
+/**
+ * Example showcasing of adding 9-patch image to style
+ * and demonstrating how it works when stretching image.
+ */
+class NinePatchImageActivity : AppCompatActivity() {
+
+  private var appendTextCounter = 1
+  private lateinit var style: Style
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_simple_map)
+
+    mapView.getMapboxMap().loadStyle(
+      styleExtension = style(Style.MAPBOX_STREETS) {
+        +image(NINE_PATCH_ID) {
+          bitmapNinePatch(BitmapFactory.decodeResource(resources, R.drawable.blue_round_nine))
+        }
+        +geoJsonSource(SOURCE_ID) {
+          feature(Feature.fromGeometry(CENTER))
+        }
+        +symbolLayer(LAYER_ID, SOURCE_ID) {
+          iconImage(NINE_PATCH_ID)
+          // make sure we stretch image both in X and Y
+          iconTextFit(IconTextFit.BOTH)
+          iconTextFitPadding(listOf(10.0, 10.0, 10.0, 10.0))
```

Comment:
BTW is this really necessary? Couldn't you accomplish padding with `imageContent` field and respective 9patch right/bottom settings?

Reply:
It could, I've changed it here just because why not, looks visually better imho.
Keep in mind it's just an example. moreover we could treat as a test that 9-patch works OK with our gl-native properties 👍